src/Filter/NumberFilter.php

```diff
@@ -19,32 +19,28 @@
 
 class NumberFilter extends Filter
 {
-    /**
-     * @param string $alias
-     * @param string $field
-     * @param string $data
-     */
```

Comment:
this should go on top of the method

Reply:
It's remove because it's [in the interface](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Filter/FilterInterface.php#L28)